druid/examples/switch.rs

```diff
@@ -29,14 +30,40 @@ fn build_widget() -> impl Widget<DemoState> {
     row.add_child(Padding::new(5.0, switch_label), 0.0);
     row.add_child(Padding::new(5.0, switch), 0.0);
 
+    let stepper = LensWrap::new(
+        Stepper::new().max(10.0).min(0.0).step(0.5).wrap(false),
+        DemoState::stepper_value,
+    );
+
+    let mut textbox_row = Flex::row();
+    let textbox = LensWrap::new(
+        Parse::new(TextBox::new()),
+        DemoState::stepper_value.map(|x| Some(*x), |x, y| *x = y.unwrap_or(0.0)),
+    );
+    textbox_row.add_child(Padding::new(5.0, textbox), 0.0);
+    textbox_row.add_child(Padding::new(5.0, stepper.center().padding(5.0)), 0.0);
```

Comment:
I'm sorry, I've led you astray. This is now adding two separate padding widgets, let's just stick with the old style and be consistent. 😅 

```suggestion
    textbox_row.add_child(Padding::new(5.0, stepper.center()), 0.0);
```

Reply:
whoops